src/api/procedures/__tests__/inviteExternalAgent.ts

```diff
@@ -155,19 +163,29 @@ describe('inviteExternalAgent procedure', () => {
 
   test('should add an add authorization transaction to the queue', async () => {
     const transaction = dsMockUtils.createTxStub('identity', 'addAuthorization');
-    const proc = procedureMockUtils.getInstance<Params, void, Storage>(mockContext, {
-      token: entityMockUtils.getSecurityTokenInstance({
-        permissionsGetAgents: [
-          {
-            agent: { did: 'otherDid' } as Identity,
-            group: entityMockUtils.getCustomPermissionGroupInstance(),
-          },
-        ],
-      }),
-    });
+    const proc = procedureMockUtils.getInstance<Params, AuthorizationRequest, Storage>(
+      mockContext,
+      {
+        token: entityMockUtils.getSecurityTokenInstance({
+          permissionsGetAgents: [
+            {
+              agent: { did: 'otherDid' } as Identity,
+              group: entityMockUtils.getCustomPermissionGroupInstance(),
+            },
+          ],
+        }),
+      }
+    );
 
+    type AuthCallback = () => AuthorizationData;
```

Comment:
used this to try to clean up the call sig a bit. I needed the transform from transform for the chaining case.